packages/ember-htmlbars/tests/helpers/input_test.js

```diff
@@ -1,492 +0,0 @@
-  owner.register('event_dispatcher:main', EventDispatcher);
-
-  var dispatcher = owner.lookup('event_dispatcher:main');
-  dispatcher.setup({}, '#qunit-fixture');
-}
-
-import isEnabled from 'ember-metal/features';
-if (!isEnabled('ember-glimmer')) {
-  // jscs:disable
-
-QUnit.module('{{input type=\'text\'}}', {
-  setup() {
-    commonSetup();
-
-    controller = {
-      val: 'hello',
-      place: 'Enter some text',
-      name: 'some-name',
-      max: 30,
-      size: 30,
-      tab: 5
-    };
-
-    view = View.extend({
-      [OWNER]: owner,
-      controller,
-      template: compile('{{input type="text" disabled=disabled value=val placeholder=place name=name maxlength=max size=size tabindex=tab}}')
-    }).create();
-
-    runAppend(view);
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-  }
-});
-
-QUnit.test('should insert a text field into DOM', function() {
-  equal(view.$('input').length, 1, 'A single text field was inserted');
-});
-
-QUnit.test('should become disabled if the disabled attribute is true', function() {
-  ok(view.$('input').is(':not(:disabled)'), 'There are no disabled text fields');
-
-  run(null, set, controller, 'disabled', true);
-  ok(view.$('input').is(':disabled'), 'The text field is disabled');
-
-  run(null, set, controller, 'disabled', false);
-  ok(view.$('input').is(':not(:disabled)'), 'There are no disabled text fields');
-});
-
-QUnit.test('input value is updated when setting value property of view', function() {
-  equal(view.$('input').val(), 'hello', 'renders text field with value');
-
-  let id = view.$('input').prop('id');
-
-  run(null, set, controller, 'val', 'bye!');
-  equal(view.$('input').val(), 'bye!', 'updates text field after value changes');
-
-  equal(view.$('input').prop('id'), id, 'the component hasn\'t changed');
-});
-
-QUnit.test('input placeholder is updated when setting placeholder property of view', function() {
-  equal(view.$('input').attr('placeholder'), 'Enter some text', 'renders text field with placeholder');
-  run(null, set, controller, 'place', 'Text, please enter it');
-  equal(view.$('input').attr('placeholder'), 'Text, please enter it', 'updates text field after placeholder changes');
-});
-
-QUnit.test('input name is updated when setting name property of view', function() {
-  equal(view.$('input').attr('name'), 'some-name', 'renders text field with name');
-  run(null, set, controller, 'name', 'other-name');
-  equal(view.$('input').attr('name'), 'other-name', 'updates text field after name changes');
-});
-
-QUnit.test('input maxlength is updated when setting maxlength property of view', function() {
-  equal(view.$('input').attr('maxlength'), '30', 'renders text field with maxlength');
-  run(null, set, controller, 'max', 40);
-  equal(view.$('input').attr('maxlength'), '40', 'updates text field after maxlength changes');
-});
-
-QUnit.test('input size is updated when setting size property of view', function() {
```

Comment:
moved to `@htmlbars Helpers test: {{input}}` : [`@test dynamic attributes`](https://github.com/emberjs/ember.js/pull/13194/files#diff-0caed08a992497ff104284ec6aea6d96R108)